src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -66,6 +66,7 @@
         "phpunit/phpunit": "<4.8.35|<5.4.3,>=5.0",
         "symfony/asset": "<3.3",
         "symfony/console": "<3.3",
+        "symfony/dependency-injection": "<=3.3-beta1",
```

Comment:
As we have a requirement on it, just update the lower bound of the requirement. It is more efficient in the composer solver than adding a conflict rule.